data/data/aws/variables-aws.tf

```diff
@@ -135,13 +135,13 @@ variable "tectonic_aws_master_root_volume_type" {
 
 variable "tectonic_aws_master_root_volume_size" {
   type        = "string"
-  default     = "30"
+  default     = "120"
```

Comment:
We don't want to be driving these defaults from the Terraform config (because that doesn't work for folks who want to install our assets themselves without going through Terraform).  Ideally we'd set it in [this structure][1] [here][2] and then push that into Terraform [here][3].  But the cluster API doesn't seem to support root volume configs at the moment (I didn't see any open issues about that, but maybe we have someone in sig-cluster-lifecycle that can ask about getting it added).  In the meantime, we're pulling this straight from the install-config, although folks that do not get this added via the install-config may not have it set in the Terraform variables at all (in which case your default here will matter).  So this probably works as you have it, but only as long as the cluster-API operator doesn't have to get involved.

[1]: https://github.com/openshift/cluster-api-provider-aws/blob/a3d9b39fbb87fdc540c63584f6daad23548bfdf8/pkg/apis/awsproviderconfig/v1alpha1/awsmachineproviderconfig_types.go#L86-L139
[2]: https://github.com/openshift/installer/blob/v0.4.0/pkg/asset/machines/aws/machines.go#L78-L100
[3]: https://github.com/openshift/installer/blob/v0.4.0/pkg/tfvars/tfvars.go#L63

Reply:
> We don't want to be driving these defaults from the Terraform config (because that doesn't work for folks who want to install our assets themselves without going through Terraform).

A good start would be to not have defaults in terraform and drive all options throught `installconfig` or `cluster-api` and slowly we move most into `not installconfig`.
https://jira.coreos.com/browse/CORS-888